src/v/cloud_topics/core/BUILD

```diff
@@ -94,6 +94,7 @@ redpanda_cc_library(
         "//src/v/bytes",
```

Comment:
Please squash with the commit which defines the write_pipeline. I guess that commit failed bazel build at all now